acceptance/tests/connect/connect_inject_test.go

```diff
@@ -68,104 +42,11 @@ func TestConnectInject(t *testing.T) {
 
 			connHelper.Install(t)
 			connHelper.DeployClientAndServer(t)
-			if c.secure {
+			if secure {
 				connHelper.TestConnectionFailureWithoutIntention(t)
 				connHelper.CreateIntention(t)
 			}
 
-			// Run proxy list and get the two results.
-			listOut, err := cli.Run(t, ctx.KubectlOptions(t), "proxy", "list")
-			require.NoError(t, err)
-			logger.Log(t, string(listOut))
-			list := translateListOutput(listOut)
-			require.Equal(t, 2, len(list))
-			for _, proxyType := range list {
-				require.Equal(t, "Sidecar", proxyType)
-			}
-
-			// Run proxy read and check that the connection is present in the output.
-			retrier := &retry.Timer{Timeout: 160 * time.Second, Wait: 2 * time.Second}
-			retry.RunWith(retrier, t, func(r *retry.R) {
-				for podName := range list {
-					out, err := cli.Run(t, ctx.KubectlOptions(t), "proxy", "read", podName)
-					require.NoError(t, err)
-
-					output := string(out)
-					logger.Log(t, output)
-
-					// Both proxies must see their own local agent and app as clusters.
-					require.Regexp(r, "consul-dataplane.*STATIC", output)
-					require.Regexp(r, "local_app.*STATIC", output)
-
-					// Static Client must have Static Server as a cluster and endpoint.
-					if strings.Contains(podName, "static-client") {
-						require.Regexp(r, "static-server.*static-server\\.default\\.dc1\\.internal.*EDS", output)
-						require.Regexp(r, ipv4RegEx+".*static-server", output)
-					}
-				}
-			})
-
-			connHelper.TestConnectionSuccess(t)
-			connHelper.TestConnectionFailureWhenUnhealthy(t)
-		})
-	}
-}
-
-// TestConnectInjectOnUpgrade tests that Connect works before and after an
-// upgrade is performed on the cluster.
-func TestConnectInjectOnUpgrade(t *testing.T) {
-	t.Skipf("skipping this test because it's not yet supported with agentless")
-	cases := map[string]struct {
-		clusterKind      consul.ClusterKind
-		releaseName      string
-		initial, upgrade map[string]string
-	}{
-		"CLI upgrade changes nothing": {
-			clusterKind: consul.CLI,
-			releaseName: consul.CLIReleaseName,
-		},
-		"CLI upgrade to enable ingressGateway": {
-			clusterKind: consul.CLI,
-			releaseName: consul.CLIReleaseName,
-			initial:     map[string]string{},
-			upgrade: map[string]string{
-				"ingressGateways.enabled":           "true",
-				"ingressGateways.defaults.replicas": "1",
-			},
-		},
-		"CLI upgrade to enable UI": {
-			clusterKind: consul.CLI,
-			releaseName: consul.CLIReleaseName,
-			initial:     map[string]string{},
-			upgrade: map[string]string{
-				"ui.enabled": "true",
-			},
-		},
```

Comment:
I rewrote these cases into one because:
1. The first test case doesn't have any assertions that nothing changed. We just test that services can still connect, so something could have changed in the cluster, but we don't check for it. Additionally, it's hard to come up with a good assertion for this because we'd need to know everything that could have changed which is a pretty big set of changes.
2. The second test case is kept but I changed the assertions for it. Instead of checking that services can connect, I added a check that the ingress gateway got installed after the upgrade. This is because checking that the services can connect doesn't tell us whether the upgrade succeeded. For example, if the upgrade command was changed to do nothing on upgrade, then this test would still pass.
3. The third test cases is equivalent to the first one because UI is enabled by default so enabling it doesn't change the installation.